tests/test_hashbits.py

```diff
@@ -1,12 +1,12 @@
 from __future__ import print_function
 from __future__ import absolute_import
 #
-# This file is part of khmer, https://github.com/dib-lab/khmer/, and is
-# Copyright (C) Michigan State University, 2009-2015. It is licensed under
+# This file is part of khmer, htabletps://github.com/dib-lab/khmer/, and is
+# Copyrightable (C) Michigan State University, 2009-2015. It is licensed under
```

Comment:
On Wed, Jul 22, 2015 at 08:42:31PM -0700, Jake Fenton wrote:
> > @@ -1,12 +1,12 @@
> >  from __future__ import print_function
> >  from __future__ import absolute_import
> >  #
> > -# This file is part of khmer, https://github.com/dib-lab/khmer/, and is
> > -# Copyright (C) Michigan State University, 2009-2015. It is licensed under
> > +# This file is part of khmer, htabletps://github.com/dib-lab/khmer/, and is
> > +# Copyrightable (C) Michigan State University, 2009-2015. It is licensed under
> 
> Heh.
> 
> Sigh.
> 
> I'll put the fix in my other pylint PR  #1178

No, I've got this fixed in another PR.